app/frontend/Calendar/EventsCell.svelte

```diff
@@ -32,8 +35,24 @@
   $: start, intervalInHours, setEnd();
   function setEnd() {
     end = new Date(start);
-    end.setHours(end.getHours() + intervalInHours);
-    displayEvents = events.filter(ev => ev.startTime >= start && ev.endTime < end);
+    if (intervalInHours % 24) {
+      end.setHours(end.getHours() + intervalInHours);
+    } else {
+      end.setDate(end.getDate() + intervalInHours / 24);
+    }
```

Comment:
not merged

@NeilRashbrook
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours
"If a parameter you specify is outside of the expected range, other parameters and the date information in the [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object are updated accordingly. For example, if you specify 100 for secondsValue, the minutes are incremented by 1 (minutesValue + 1), and 40 is used for seconds."

1. Based on that, do you still think this change is necessary?
2. Could you please test whether the bug (which you tried to fix here) still appears on master? (master does not have this change)

Reply:
I can't remember why I might have made this change; I tried testing what I thought it might have been and it wasn't that, so you're probably right and this change isn't necessary.